src/middlewared/middlewared/api/v25_04_0/smartctl.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any
+
+from middlewared.api.base import BaseModel
+
+__all__ = ["AtaSelfTest", "NvmeSelfTest", "ScsiSelfTest"]
+
+class AtaSelfTest(BaseModel):
+    num: int
+    description: str
+    status: str
+    status_verbose: str
+    remaining: float
+    lifetime: int
+    lba_of_first_error: int | None = None
+
+class NvmeSelfTest(BaseModel):
+    num: int
+    description: str
+    status: str
+    status_verbose: str
+    power_on_hours: int
+    failing_lba: int | None = None
+    nsid: int | None = None
+    seg: int | None = None
+    sct: int | None = 0x0
```

Comment:
Why is it nullable?

Reply:
For these instances where these fields in the schema are nullable, it's because those keys only show up in specific circumstances in the JSON output. Usually related to if a test is running, and more usually if a test has failed.